examples/holiday_of_day.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
```

Comment:
Given how similar this one and the other two look, it might be advantageous to combine shared code in a separate module but also not a big deal right now.